.travis.yml

```diff
@@ -2,5 +2,7 @@ sudo: required
 language: node_js
 node_js:
   - "0.10"
-before_install:
- - sudo apt-get update -qq
+after_success:
+ - ./node_modules/.bin/istanbul cover -x "**/spec/**" ./node_modules/.bin/_mocha --report lcovonly -- $(find spec -name '*-spec.js') -R spec --require spec/helper.js
+ - cat ./coverage/lcov.info | node_modules/.bin/coveralls
+
```

Comment:
I think it would be better to put this command into the package json as an npm script, so that in the travis file we can just say something like npm coverage